phing/tasks/acsf.xml

```diff
@@ -10,10 +10,18 @@
     <phingcall target="acsf:init:drush"/>
   </target>
 
+  <!-- Perform acsf:init for each multisite. -->
+  <target name="acsf:init:drush">
+    <!-- Get the mulitsite names. -->
+    <yamlKeys file="${blt.config-files.project}" variable="multisite" outputProperty="multisite.names"/>
+    <foreach list="${multisite.names}" param="multisite.name" target="acsf:init:site:drush"/>
```

Comment:
I didn't realize that acsf had to be initialized per multisite. @nikgregory any concerns here?